packages/calypso-e2e/src/lib/flows/start-import-flow.ts

```diff
@@ -61,7 +62,8 @@ export class StartImportFlow {
 	 * Validates that we've landed on the setup page.
 	 */
 	async validateSetupPage(): Promise< void > {
-		await this.page.waitForSelector( selectors.startImportButton );
+		( await this.page.locator( selectors.startImportButton ) ) ||
+			( await this.page.locator( selectors.startImportGoalButton ) );
```

Comment:
Using `||` to wait on multiple selectors likely does not behave in the way that's expected.

FYI, Playwright supports waiting for [multiple CSS selectors](https://playwright.dev/docs/selectors#selecting-elements-matching-one-of-the-conditions) and I would suggest doing that instead. It would look something like this:

```
await this.page.locator( `${selector1}, ${selector2}` )
```

Alternatively, you can also try using `await Promise.race`.